packages/spark-core/setup-core.scss

```diff
@@ -0,0 +1,62 @@
+///
+// Generic utility styles
+///
+
+@import "../@scalescss/generic-normalize/normalize";
+@import "../@scalescss/generic-reset/reset";
+
+///
+// Spark-core settings
+///
+@import './settings/colors';
```

Comment:
I'd move this and settings above everything else since they don't output any code themselves. Also, reset has options that can be replaced by settings.